langwatch/prisma/migrations/20250210154735_add_annotation_queue_item/migration.sql

```diff
@@ -0,0 +1,42 @@
+/*
+  Warnings:
+
+  - A unique constraint covering the columns `[projectId,slug]` on the table `AnnotationQueue` will be added. If there are existing duplicate values, this will fail.
+  - Added the required column `slug` to the `AnnotationQueue` table without a default value. This is not possible if the table is not empty.
+
+*/
```

Comment:
that's a good point, do we have any annotation queues right now on the production db? If so we need to delete it before merging this